content/millennium/r4/individuals/related-person.md

```diff
@@ -55,11 +58,13 @@ Search for RelatedPersons that meet supplied query parameters:
 
 ### Parameters
 
- Name         | Required?                                         | Type          | Description
---------------|---------------------------------------------------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------
- `_id`        | This or `patient` or `identifier` | [`token`]     | The logical resource id associated with the resource.
- `identifier` | This or `_id` or `patient`        | [`token`]     | A RelatedPerson identifier.
- `patient`    | This or `_id` or `identifier`     | [`reference`] | A reference to a patient associated with the RelatedPerson. Example: `14067892`
+ Name                  | Required?                                                | Type          | Description
+-----------------------|----------------------------------------------------------|---------------|---------------------------------------------------------------------------------------------------------------
+ `_id`                 | This or `patient` or `identifier` or `-encounter`        | [`token`]     | The logical resource id associated with the resource.
+ `identifier`          | This or `_id` or `patient` or `-encounter`               | [`token`]     | A RelatedPerson identifier.
+ `patient`             | This or `_id` or `identifier` or `-encounter`            | [`reference`] | A reference to a patient associated with the RelatedPerson. Example: `14067892`
+ `-encounter`          | This or `_id` or `patient` or `identifier`               | [`reference`] | A reference to an encounter associated with the RelatedPerson. Example: `97697434`
+ `-relationship-level` | This can only be provided with `patient` or `identifier` | [`token`]     | The resource’s relationship to patient. Example: `-relationship-level= http://hl7.org/fhir/resource-types|Patient`
```

Comment:
Do we need to alphabetize these?

Reply:
I don't believe so, I think they are currently in order of the spec and then our custom extensions.